styles/globals.css

```diff
@@ -10,7 +10,6 @@ body {
     max-width: 100%;
     overflow-x: hidden;
     font-size: 1rem;
-    scroll-behavior: smooth;
```

Comment:
**question** why remove this property ?

Reply:
It is duplicated and is above. This applies to all the rules that have been corrected